docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -99,13 +99,12 @@ You can customize these for your own needs.
 
 To configure discovery, follow these steps:
 
-. In the web UI, click the *Gear* symbol in the top-right of the screen.
-. Choose menu:Provisioning[Configure Discovery].
+. Click the *Gear* symbol in the top-right of the screen.
+. Under Provisioning, click *Configure Discovery*.
 . In the *General Settings* area, select the requisition you just created from the *Requisition* list.
 ** (Optional) Change the default values.
 . Click *Save and Restart Discovery* in the top-left of the screen.
-. Return to *Manage Provisioning Requisitions* and click *Synchronize the Requisition*.
+. Return to *Manage Provisioning Requisitions* and click *Synchronize the Requisition* (image:provisioning/sync_requisition.png[synchronize requisition symbol,20]).
```

Comment:
Does this render with no issues if the image is on the same line? Do we want to force a line break?

Reply:
It actually did. I thought we couldn't do inline images, but it actually works. Had to play with the resizing. But to adhere to our one sentence/element per line, we should probably force a line break. I'll build locally and see if it still renders correctly.

Reply:
Unfortunately, with the forced line break puts the graphic on the next line instead of inline.

Reply:
Ah, that sucks. It was worth a try!